classes/class-swsales-reports.php

```diff
@@ -15,6 +15,7 @@ public static function init() {
 		add_action( 'wp_enqueue_scripts', array( __CLASS__, 'enqueue_tracking_js' ) );
 		add_action( 'wp_ajax_swsales_ajax_tracking', array( __CLASS__, 'ajax_tracking' ) );
 		add_action( 'wp_ajax_nopriv_swsales_ajax_tracking', array( __CLASS__, 'ajax_tracking' ) );
+		add_action("wp_ajax_sws_stats_csv", array(__CLASS__, "sws_stats_csv"));
```

Comment:
Are we pretty consistent about using swsales_ as the prefix?

If so, let's update the AJAX call and related callback function names to swsales_...

Reply:
Removed prefix in class method; updated prefix for ajax action.